### ISSUE TITLE
Progressive mode for crafting guide

### ISSUE THREAD

**Issue by arunisaac** [minetest-LOTR/Lord-of-the-Test#151]
Hi,

This is a feature request. Can we have a progressive mode for the crafting guide whereby only recipes for items you have ever had in your inventory are shown? Perhaps this should even be the default.

Thanks!

**Comment by DonFlymoor** [comment]
I agree that this should be a feature - but that it should be taken a step further. Some recipes you should be able to figure out on your own, but the harder ones should have to be learned from NPCs, who may teach you how to make an item for money, completing a quest, or just because you asked nicely.

**Comment by arunisaac** [comment]
Yes, that sounds great! I like the way your suggestion enhances the RPG
element.
